### PR TITLE
Make wiki/tag revisions voteable

### PR DESCRIPTION
**Pull request by jimrandomh**
Makes wiki/tag revisions voteable. Not really tested and probably still quite buggy. Doesn't yet handle the automatic self-upvote, which needs a callback and a migration. Karma-change notifications are theoretically handled, but not tested yet.

### CODE REVIEW

**Comment by Discordius** [comment]
Let me know whether you want me to review it, or what the best way for me to interact with this is. Might just be "ignore it until it's ready" or might be "give input on basic architecture"

**Comment by jimrandomh** [comment]
Probably "ignore it until it's ready".

**Comment by jimrandomh** [comment]
Not exactly battle tested, but tested as far as creating a tag, having another user upvote it, and seeing the correct karma score on the All Posts page and in the karma-change notifier. So, ready to review.

Known issues that don't seem bad enough to not ship:
 * At the largest breakpoint, the vote-count is vertically misaligned relative to the vote buttons
 * The links in the karma-change notifier go to the tag history page, rather than something specific to the individual revision

File: packages/lesswrong/components/votes/SmallSideVote.tsx [hunk 1]
```diff
@@ -55,20 +57,31 @@ const SmallSideVote = ({ document, hideKarma=false, classes, collection }: {
   const karma = voteProps.baseScore;
 
   let moveToAlignnmentUserId = ""
+  let documentTypeName = "comment";
```
Comment (Discordius) [inline packages/lesswrong/components/votes/SmallSideVote.tsx:60]: Hmm, not a huge fan of doing this with a mutable variable. Felt like it made it harder to read (as opposed to a switch or even just two ternary statements). But doesn't seem that terrible.

**Review by Discordius — APPROVED** [review]

**Pull request merged**